### ISSUE TITLE
Editing and saving subsections

### ISSUE THREAD

**Issue by MrReSc** [turnermm/ckgedit#487]
### Saving subsections
If you have several identical headings on one page (e.g. Blafoo [h3]) and you edit one (...#blafoo3) of these sections on this page with CKGedit and then save it, you jump back to the first section (...#blafoo) with the same heading on the page after saving. It looks like it's just looking at the headline in the Edior.

![2019-07-16_06-58-18](https://user-images.githubusercontent.com/24952268/61267210-3f92db00-a797-11e9-9f93-78e3f040cc2f.gif)

### Editing and canceling subsections
If I edit a section, even if it has a unique heading, and then click Cancel instead of Save, then I jump to the top of the page.

![2019-07-16_07-02-48](https://user-images.githubusercontent.com/24952268/61267350-dcee0f00-a797-11e9-83e7-dfa61356cf01.gif)

Both described situations work with the DW Editor as expected.

**Comment by turnermm** [comment]
Are you saying it saves incorrectly and loses the edits.

As a side-note:  I don't find those animated gifs helpful.  They are hard to follow, and I don't know what I'm looking at.

**Comment by turnermm** [comment]
If it's what I think you are saying, it's a nice feature, and I'll take it as that--a feature request and not as a critical error.

**Comment by turnermm** [comment]
And in the end I  might not be able to do anything about it.

**Comment by turnermm** [comment]
It turned out to be a reasonably simple fix.  Se the latest update.

**Comment by MrReSc** [comment]
Sorry about my late answer.
Everything will be saved correctly, don't worry.

It's just a "comfort function" which is very helpful for large pages. Some users have pointed this out to me.

Regarding GIF's, since my native language is not English, this problem was not easy to describe. Sometimes it is easier to show something.

As soon as I have time, I will test the update. Is it already available through Plugin Manger?

**Comment by turnermm** [comment]
> As soon as I have time, I will test the update. Is it already available through Plugin Manger?

Yes.  It seems to be working nicely, both for cancel and save.
